mail_footer_notified_partners/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Mail Footer Notified Partners
+=============================
+
+This module adds into the footer the partner's name notified by this email
```

Comment:
This module adds the list of notified partners in the footer of notification e-mails sent by Odoo.